### DISCUSSION TITLE
How to extract "pr std dev" from ubx file?

### DISCUSSION THREAD

**Discussion by hongzhaozheng** [semuconsulting/pyubx2#85] [Q&A]
Hi,

Does anyone know how to extract "PR std dev" from UBX-RXM-RAWX in u-center or just from a .ubx file?

Hongzhao

**Comment by semuadmin** [comment]
Hi @hongzhaozheng 

Unfortunately pyubx2 doesn't currently decode RAWX data and there  are no plans to add this functionality. 

This would require the aggregation of time series data over multiple UBX-RXM-RAWX messages (25 messages or more), whereas pyubx2 is designed as a 'single-pass' parser, decoding individual UBX messages into single UBXMessage objects.
